src/Handler/EasyHandle.php

```diff
@@ -23,7 +23,7 @@ final class EasyHandle
     /** @var array Received HTTP headers so far */
     public $headers = [];
 
-    /** @var ResponseInterface Received response (if any) */
+    /** @var ?ResponseInterface Received response (if any) */
```

Comment:
`ResponseInterface|null`

Reply:
fixed in 583f8291db35249b94ca8893068002a654caf4a1